### ISSUE TITLE
Settings Page: Adding Functionalities

### ISSUE THREAD

**Issue by sreejakshetty** [fossasia/engelsystem#218]
https://github.com/fossasia/engelsystem/issues/165#issuecomment-238605895
- [x] Fix the initialisation error.

```
Notice: Undefined variable: ok in engelsystem/includes/pages/admin_settings.php on line 58
Notice: Undefined variable: msg in engelsystem/includes/pages/admin_settings.php on line 68
Notice: Undefined variable: event_name in engelsystem/includes/pages/admin_settings.php on line 74
Notice: Undefined variable: buildup_start_date in engelsystem/includes/pages/admin_settings.php on line 75
Notice: Undefined variable: event_start_date in engelsystem/includes/pages/admin_settings.php on line 76
Notice: Undefined variable: event_end_date in engelsystem/includes/pages/admin_settings.php on line 77
Notice: Undefined variable: teardown_end_date in engelsystem/includes/pages/admin_settings.php on line 78
Notice: Undefined variable: event_welcome_msg in engelsystem/includes/pages/admin_settings.php on line 80
Notice: Undefined variable: ok in engelsystem/includes/pages/admin_settings.php on line 58
Notice: Undefined variable: msg in engelsystem/includes/pages/admin_settings.php on line 68
```
- [x] validate buildup_start_date <= event_start_date <= event_end_date <= teardown_end_date
- [ ] Display the entered data: Event-name and the dates on login and news (start and home screen) and the welcome message on registration (or whatever your idea was). I think of a 3 column layout for the login page (1. column is event data, 2. column is login, 3. column is the miscellaneous links and infos)

**Comment by sreejakshetty** [comment]
> I think of a 3 column layout for the login page (1. column is event data, 2. column is login, 3. column is the miscellaneous links and infos)

New login page layout suggested by @msquare.
![screenshot from 2016-08-17 16 15 48](https://cloud.githubusercontent.com/assets/14835922/17733817/1c5ae278-6496-11e6-8e2a-962a2e31cb3c.png)

**Comment by mariobehling** [comment]
@kamishettysreeja25 There is probably not enough time to implement this before the end of GSoC and this feature needs to be thought-through more. The left sidebar does not work from a design perspective.

**Comment by sreejakshetty** [comment]
@mariobehling 
Yes, we need to work on how the design should be. We can postpone it for future milestones. I have just implemented a sample design to show how does it look if we divide login page to three column layout.
Thank you.

**Comment by sreejakshetty** [comment]
Fixed here #224
